src/views/HomePage.vue

```diff
@@ -436,14 +439,18 @@ ul.custom-bullet li::before {
     height: 7vh;
   }
   .footer {
-    font-size: 0.9rem;
-    bottom: 8px;
+    font-size: 0.8rem;
+    bottom: 4px;
+  }
+  .ion-toolbar-header {
+    border: 1px solid red;
+    padding: 0px;
   }
 }
 
 @media (max-width: 380px) {
```

Comment:
Update the max width for this media query

Reply:
done